src/lib/definitions.js

```diff
@@ -895,6 +895,8 @@ function getFHPaddingEntries(index)
             define('escape([[]][+(RP_0_S + AT)[0]] + AT)[61]', AT, NO_FF_SRC), // *
             define('escape([NaN][+(RP_1_WA + AT)[20]] + AT)[61]', AT, NO_IE_SRC), // *
             define('escape(true + AT)[50]', AT, V8_SRC),
+            define('(RP_3_WA + "".slice.call())[11]', CALL_ON_GLOBAL, DOMWINDOW),
+            define('"".sub.call()[13]', CALL_ON_GLOBAL, DOMWINDOW),
```

Comment:
I think that these two definitions are always worse than this one:

https://github.com/fasttime/JScrewIt/blob/d9e7ef2de5b339df81c604f479011e2ec0529302/src/lib/definitions.js#L886